src/stylesheets/App.css

```
@@ -119,8 +119,9 @@ a:hover {
 .submit-btn {
   font-weight: bold;
   color: #ffffff;
-  background-color: #acd157;
+  background-color: transparent;
   opacity: 0.5;
+  border: white;
```

we should set width and border style here

```suggestion
  border: 2px solid #ffffff;
```